x/capability/types/keys_test.go

```diff
@@ -43,7 +43,7 @@ func TestFwdCapabilityKeyCompatibility(t *testing.T) {
 	cap := types.NewCapability(24)
 	new := types.FwdCapabilityKey("bank", cap)
 	old := legacyFwdCapabilityKey("bank", cap)
-	if runtime.GOOS == "darwin" && runtime.GOARCH == "arm" {
+	if runtime.GOOS == "darwin" && (runtime.GOARCH == "arm" || runtime.GOARCH == "arm64") {
```

Comment:
Is it okay to remove GOOS condition? becasue this condition can be hit on arm64/linux, not only m1 mac